packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/graphql/GraphQL-ConnectedDemo.dynamic.graphql

```diff
@@ -0,0 +1,69 @@
+# This file contains a GraphQL query that will be executed and the result provided to
```

Comment:
Remove these GraphQL components (these are all duplicates of the Styleguide ones, right)?